airbyte-ci/connectors/pipelines/pipelines/cli/click_decorators.py

```diff
@@ -121,3 +126,27 @@ def decorated_function(*args: Any, **kwargs: Any) -> Any:  # noqa: ANN401
             return f(*args, **kwargs)
 
         return decorated_function
+
+
+def click_ci_requirements_option() -> Callable[[FC], FC]:
```

Comment:
We can eventually add arguments to this decorator to customize requirements according to commands.